iwinfo_cli.c

```diff
@@ -327,6 +327,17 @@ static char * format_assocrate(struct iwinfo_rate_entry *r)
 			p += snprintf(p, l, ", HE-DCM %d", r->he_dcm);
 			l = sizeof(buf) - (p - buf);
 		}
+		else if (r->is_eht)
+		{
+			p += snprintf(p, l, ", EHT-MCS %d, %dMHz", r->mcs, r->mhz);
```

Comment:
Assuming the updated structure, this will become:
`p += snprintf(p, l, ", EHT-MCS %d, %dMHz", r->mcs, r->mhz_hi * 256 + r->mhz);`